concepts/05 Widgets/Form/01 Getting Started with Form/20 Populate the Form Fields.md

```diff
@@ -0,0 +1,126 @@
+The **Form** generates a [simple item](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/) for each field in the [formData]() object. A simple item is a label-editor pair. 
+
+To populate form fields, assign an object to the [formData](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#formData) property. Each field in this object will be visualized as a label-editor pair. Such pairs are called "simple items". The **Form** automatically defines which editors it will generate for fields in the **formData** object. The editor's type depends on the data type of the field's value. The **Form** in the following example will create the [TextBox](/Documentation/ApiReference/UI_Widgets/dxTextBox/), [NumberBox](/Documentation/ApiReference/UI_Widgets/dxNumberBox/), and [DateBox](/Documentation/ApiReference/UI_Widgets/dxDateBox/) editors:
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#form").dxForm({
+            formData: {
+                name: "John Heart",
+                officeNumber: 901,
+                hireDate: new Date(2012, 4, 13)
+            }
+        });
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-form
+        [formData]="employee">
+    </dx-form>
+
+    <!-- tab: app.component.ts -->
+    import { Component } from '@angular/core';
+
+    @Component({
+        selector: 'app-root',
+        templateUrl: './app.component.html',
+        styleUrls: ['./app.component.css']
+    })
+    export class AppComponent {
+        employee = {
+            name: "John Heart",
+            officeNumber: 901,
+            hireDate: new Date(2012, 4, 13)
+        }
+    }
+
+    <!-- tab: app.module.ts -->
+    import { BrowserModule } from '@angular/platform-browser';
+    import { NgModule } from '@angular/core';
+    import { AppComponent } from './app.component';
+
+    import { DxFormModule } from 'devextreme-angular';
+
+    @NgModule({
+        declarations: [
+            AppComponent
+        ],
+        imports: [
+            BrowserModule,
+            DxFormModule
+        ],
+        providers: [ ],
+        bootstrap: [AppComponent]
+    })
+    export class AppModule { }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div id="app-container">
+            <DxForm 
+                :form-data="employee">
+            </DxForm>
+        </div>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxForm } from 'devextreme-vue/form';
+
+    let employee = {
```

Comment:
const?

Reply:
If we modify the form data at runtime, won't this object change?